app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,59 @@
+class CustomersController < ApplicationController
+  def index
+    customers = Customer.all
+    render json: customers, status: :ok
+  end
+
+  def overdue
+    # overdue_rentals = Rental.where("movie_id = ? AND customer_id = ?", movie.id, customer.id)
+    # theoretical: has_overdue?
+    customers_with_overdue= Customer.all.select {|customer| customer.has_overdue?}
```

Comment:
While this will work, it's usually better to let the database do a lot of the work, with some sort of `where` clause, probably in a class method of `Customer` which selects movies which are overdue.

The database is almost always faster about this.